.github/workflows/tf-account.yaml

```diff
@@ -37,19 +50,34 @@ jobs:
         with:
           terraform_version: ${{ inputs.terraform-version }}
 
+      - name: Tailscale
+        uses: tailscale/github-action@v2
+        # a tailscale oauth client requires tags
+        if: inputs.tailscale-tags != ''
+        with:
+          oauth-client-id: ${{ secrets.TS_OAUTH_CLIENT_ID }}
```

Comment:
Leaving as is for now since it matches all the tailscale docs.

Reply:
I don't think they made a considered choice. I'm anti-abbreviations because of the impact on discoverability. If you Google this identifier without double quotes you get a bunch of TypeScript stuff.

Their own docs aren't even consistent:

https://tailscale.com/kb/1276/tailscale-github-action/